### PR TITLE
make the Auth service the primary login provider on staging

### PR DESCRIPTION
**Pull request by bystam**
# Jira Issue: []

## What?
- Add the missing bits to use the auth service alongside gatekeeper
- Make the auth service default on staging

## Why?
- We want to roll it out on staging first.

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,6 +2,7 @@ OAUTH_CLIENT_ID=<a uuid>
 OAUTH_CLIENT_SECRET=<very secure>
 API_URL=http://localhost:8443
 GATEKEEPER_HOST=<optional gatekeeper host>
+AUTH_HOST=<optional auth host>
```
Comment (bystam) [inline .env.example:5]: I guess this needs to be added somewhere?

File: src/portals/sos/pages/MainPage.tsx [hunk 2]
```diff
@@ -28,21 +28,26 @@ const Footer = styled.div`
...
+        .hex()};
     margin-top: 4vh;
     transition: color 200ms;
 
     :hover {
-      color: ${({ theme }) => chroma(theme.foreground).brighten(2).hex()};
+      color: ${({ theme }) =>
+        chroma(theme.foreground)
+          .brighten(2)
+          .hex()};
```
Comment (bystam) [inline src/portals/sos/pages/MainPage.tsx:42]: This must've been auto-formatted through a git hook?

Comment (rasmusguterstam) [reply src/portals/sos/pages/MainPage.tsx:42]: Yeah I think there's a pre-commit prettify-something

File: src/entry.tsx [hunk 3]
```diff
@@ -47,14 +47,12 @@ ReactDOM.render(
...
+              path="/login-provider"
               component={() => {
                 window.location.href = `${
-                  (
-                    window as Window &
-                      typeof global & { GATEKEEPER_HOST: string }
-                  ).GATEKEEPER_HOST
-                }/sso?redirect=${window.location.protocol}//${
+                  (window as Window & typeof global & { LOGIN_URL: string })
+                    .LOGIN_URL
```
Comment (rasmusguterstam) [inline src/entry.tsx:54]: I'm guessing this?

```suggestion
                    .AUTH_HOST
```

Comment (rasmusguterstam) [reply src/entry.tsx:54]: Aah nvm, saw more 🤓

File: .env.example (hunk 1 above)
Comment (rasmusguterstam) [reply .env.example:5]: Would it be controversial to just add the staging auth host here directly? This repo is public though, but it's not like we're exposing secrets 😄

**Review by rasmusguterstam — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (bystam) [reply .env.example:5]: Yeah, I will.

**Pull request merged**